spec/spec_helper.rb

```diff
@@ -1,7 +1,14 @@
 # frozen_string_literal: true
 
-require 'bundler/setup'
+require "bundler/setup"
+require "combustion"
+require "rails"
+require "active_record/railtie"
+require "action_controller/railtie"
+Bundler.require :default, :development
 Bundler.setup
+Combustion.initialize! :all
+# spec/spec_helper.rb
 
 require 'ready_set'
```

Comment:
@helpotters I think we need to call `Combustion.initialize!` with our railtie -- this seems to make the railtie spec pass:
```suggestion
require 'ready_set'
Bundler.require :default, :development
Bundler.setup
Combustion.initialize! 'ready_set'
# spec/spec_helper.rb
```